components/frontend/src/dashboard/StatusBarChart.js

```diff
@@ -33,18 +35,15 @@ export function StatusBarChart({ animate, colors, label, tooltip, summary, maxY,
     bars.reverse();
     colors.reverse();
     return (
-        <svg viewBox="0 0 400 400">
```

Comment:
Wasn't this needed for tags under the cards, when the screen space is very narrow?

Reply:
This is gone.